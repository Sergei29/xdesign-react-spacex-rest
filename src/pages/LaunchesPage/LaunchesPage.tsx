import React from "react";

const LaunchesPage = () => {
  return <div></div>;
};

export default LaunchesPage;
