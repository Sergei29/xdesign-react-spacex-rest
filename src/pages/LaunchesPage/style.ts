import styled from "styled-components";

export const LaunchesPageContainer = styled.section``;
