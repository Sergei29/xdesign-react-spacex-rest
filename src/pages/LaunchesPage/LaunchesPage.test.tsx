import React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import LaunchesPage from "./LaunchesPage";

describe("LaunchesPage", () => {
  let wrapper: ShallowWrapper;
  beforeEach(() => {
    wrapper = shallow(<LaunchesPage />);
  });

  it("should render", () => {
    expect(wrapper.length).toEqual(1);
  });

  it("should match snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
