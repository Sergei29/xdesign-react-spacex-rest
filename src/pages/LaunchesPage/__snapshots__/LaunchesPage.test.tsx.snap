// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LaunchesPage should match snapshot 1`] = `ShallowWrapper {}`;
